wgsl/index.bs

```diff
@@ -5826,16 +5826,140 @@ these functions must only be invoked in uniform control flow.
 
 ### Arrayed resource access TODO ### {#arrayed-resource-access}
 
-## Floating Point Evaluation TODO ## {#floating-point-evaluation}
+## Floating Point Evaluation ## {#floating-point-evaluation}
+
+[SHORTNAME] follows the IEEE 754 standard for floating point computation with
+the following exceptions:
+* No floating point exceptions are generated.
+* Signaling NaNs may not be generated.
+    Any signaling NaN may be converted to a quiet NaN.
+* Implementations may assume that NaNs, inifinities are not present
+    * Note: This means some functions (e.g. `isInf`, `isNan`, `min` and `max`)
+        may not return the expected result due to optimizations about the presence
+        of NaNs and infinities.
+* Implementations may ignore the sign of a zero.
+    That is, a zero with a positive sign may behave like a zero a with a negative sign, and vice versa.
+* No rounding mode is specified.
+* Implementations may flush denormalized value on the input and/or output of
+    any operation listed in [[#floating-point-accuracy]].
+    * Other operations are required to preserve denormalized numbers.
+* The accuracy of operations is given in [[#floating-point-accuracy]].
+
+### Floating Point Accuracy ### {#floating-point-accuracy}
+
+An operation is <dfn noexport>correctly rounded</dfn> if its result is a
+floating point number nearest to the exact result computed with infinite
+precision.
+The rounding mode is unspecified for [SHORTNAME].
+If the result, `x`, is exactly representable in floating point, then `x` must be
+returned.
+Otherwise, a floating point value closest to `x`, either no less than `x`
```

Comment:
I don't understand the "either not less or no greater than". Does it have a specific meaning here?

Reply:
Hm.  Right, that phrase seems to provide no value.

I checked the IEEE 754 wording and it's pretty straightforward. Mostly it deals with the different prescribed rounding modes, but WGSL doesn't specify rounding mode or a way to set it.
So I think this sentence can be simplified.

>  Otherwise, the result is a floating point value closest to `x`.

and maybe: 

> Note: When two floating point values are equally close to `x`, the result can be either.